source/users/delete-or-revoke.txt

```diff
@@ -91,6 +103,66 @@ Manually Delete a User
 
 .. include:: /includes/note-delete-user.rst
 
+Delete a User with a Custom Function
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can write a custom function to delete a user. You might want to do this if
+your SDK does not yet support the delete users API. 
+
+:ref:`Create a function <define-a-function>` similar to our example below 
+that uses :guilabel:`Application Authentication`.
+
+For this example function, we have :ref:`created secrets for the 
+adminApiPublicKey and adminApiPrivateKey <values-and-secrets>`. 
+We would then add the :ref:`Project and Application IDs 
+<find-your-project-or-app-id>` to the ``apiUrl``.
+
+We can then :ref:`call this function from the SDK 
+<call-function-from-client-sdks>`, passing in the ``user.id``.
+
+.. code-block:: javascript
+
+   const apiUrl = "https://realm.mongodb.com/api/admin/v3.0/groups/{insert-your-groupId-here}/apps/{insert-your-appId-here}";
+
+   // deleteAUser() deletes a single user from the database using the Admin API 
+   // pass in an array with an object with the key userID (representing the app user's user.id)
+   // return value if successful = id of the deleted user
```

Comment:
This seems... very dangerous. Can we add a check so that a given user can only delete their own account? Otherwise, we're creating a function that anyone can use to destroy anyone else's account. I would probably make this function uncallable from the SDKs, and call it from another function that checks the calling user ID.

Unless I'm missing something here, in which case... fair enough!